activemq/rootfs/etc/confd/templates/activemq.xml.tmpl

```diff
@@ -0,0 +1,158 @@
+                        </defaultEntry>
+                    </redeliveryPolicyMap>
+                </redeliveryPolicyMap>
+            </redeliveryPlugin>
+        </plugins>
+
+        <destinationPolicy>
+            <policyMap>
+              <policyEntries>
+                <policyEntry topic=">" >
+                    <!-- The constantPendingMessageLimitStrategy is used to prevent
+                         slow topic consumers to block producers and affect other consumers
+                         by limiting the number of messages that are retained
+                         For more information, see:
+
+                         http://activemq.apache.org/slow-consumer-handling.html
+
+                    -->
+                  <pendingMessageLimitStrategy>
+                    <constantPendingMessageLimitStrategy limit="1000"/>
+                  </pendingMessageLimitStrategy>
+                </policyEntry>
+                <policyEntry queue=">">
+                  <deadLetterStrategy>
+                      <!-- ActiveMQ.DLQ will keep persistent, undelivered messages for one week -->
+                      <sharedDeadLetterStrategy expiration="{{ getv "/dlq/expiration/ms" "604800000" }}" processExpired="{{ getv "/dlq/process/expired" "true" }}" processNonPersistent="{{ getv "/dlq/process/non/persistent" "true" }}" />
+                  </deadLetterStrategy>
+                </policyEntry>
+              </policyEntries>
+            </policyMap>
+        </destinationPolicy>
+
+
+        <!--
+            The managementContext is used to configure how ActiveMQ is exposed in
+            JMX. By default, ActiveMQ uses the MBean server that is started by
+            the JVM. For more information, see:
+
+            http://activemq.apache.org/jmx.html
+        -->
+        <managementContext>
+            <managementContext createConnector="false"/>
+        </managementContext>
+
+        <!--
+            Configure message persistence for the broker. The default persistence
+            mechanism is the KahaDB store (identified by the kahaDB tag).
+            For more information, see:
+
+            http://activemq.apache.org/persistence.html
+        -->
+        <persistenceAdapter>
+            <kahaDB directory="${activemq.data}/kahadb"/>
+        </persistenceAdapter>
+
+
+          <!--
+            The systemUsage controls the maximum amount of space the broker will
+            use before disabling caching and/or slowing down producers. For more information, see:
+            http://activemq.apache.org/producer-flow-control.html
+          -->
+          <systemUsage>
+            <systemUsage>
+                <memoryUsage>
+                    <memoryUsage percentOfJvmHeap="70" />
+                </memoryUsage>
+                <storeUsage>
+                    <storeUsage limit="100 gb"/>
+                </storeUsage>
+                <tempUsage>
+                    <tempUsage limit="50 gb"/>
```

Comment:
I'd imagine the `<storageUsage>` is where our persistent storage is mounted.  If `<tempUsage>` refers to /tmp then 50GB is too high as the kubernetes nodes only have 20GB of ephemeral storage which is shared across all the running containers.  And when that space starts to fill up, the kubernetes scheduler will find the container using the most and evict it.

Reply:
Good catch.  I have a commit that parameterizes those options and sets a default for `<tempUsage>` to 18 gb.  Waiting for images to build and tests to succeed.

Reply:
Looks great!